lib/perl/Genome/VariantReporting/Suite/BamReadcount/VafInterpreterHelpers.t

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env genome-perl
+
+BEGIN { 
+    $ENV{UR_DBI_NO_COMMIT} = 1;
+    $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+}
+
+use strict;
+use warnings;
+
+use above "Genome";
+use Test::More;
+use List::Util qw(shuffle);
```

Comment:
I'm not sure about using shuffle here.  Perhaps if we repeated the test a number of times we could guarantee that we didn't just accidentally pass it.  Does shuffle guarantee that the order isn't coincidentally the same as the input order?